css/block.css

```diff
@@ -233,7 +233,7 @@ wb-value[type*=rect] > input{background-image: url(../images/icon/rect.svg);}
 wb-value[type*=string] > input{background-image: url(../images/icon/string.svg);}
 wb-value[type*=geolocation] > input{ background-image: url(../images/icon/geolocation.svg);}
 wb-value[type*=size] > input{ background-image: url(../images/icon/size.svg);}
-wb-value[type*=date] > input{ background-image: url(../images/icon/calendar.svg);}
```

Comment:
We could go without it but the icon in the wb-value in the menu is a little large and extends the input box past the bounds of the block